bin/main.py

```diff
@@ -5,34 +5,48 @@
   main.py backup
   main.py restore
   main.py init
+  main.py log --list [--limit=<number>]
+  main.py log [--version=<version>]
```

Comment:
Would this second form would be more idiomatic as `main.py log [<version>]`?

Reply:
Or even as `main.py log list` or `main.py log show [<version>]` as the two forms

Reply:
Also, the name `main.py` should probably be bb8 so that running `bb8 --help` gives more understandable output

Reply:
nice!